### PR TITLE
Joining the Windows Insider Program

### PR DESCRIPTION
**Pull request by cleverActon0126**
Added a new wiki, this page REQUIRES monthly maintenance to maintain the update. Please feel free to contribute to it.

### CODE REVIEW

**Comment by Erisa** [comment]
>  this page REQUIRES monthly maintenance to maintain the update. 

What do you mean?

**Comment by cleverActon0126** [comment]
Just to remind myself to keep it updated. You can ignore that.

**Comment by goldsolitude** [comment]
Looks okay, some images are not correct for _Leaving the Windows Insider Program_ section however and there are a couple grammatical errors but that can be fixed while its being finalized. The _Error Code_ section in troubleshooting should be moved to the bottom as it doesn't contain much content and looks odd between two filled sections,

**Comment by cleverActon0126** [comment]
Alright, I will work on this in a bit. Thanks :D

> Looks okay, some images are not correct for _Leaving the Windows Insider Program_ section however and there are a couple grammatical errors throughout the entire document but that can be fixed while its being finalized. The _Error Code_ section in troubleshooting should be moved to the bottom as it doesn't contain much content and looks odd between two filled sections,

**Comment by cleverActon0126** [comment]
Also can you please point out the image that is not correct for leaving? My end is kinda messed up now since I did this so many times.

**Comment by goldsolitude** [comment]
> Also can you please point out the image that is not correct for leaving? My end is kinda messed up now since I did this so many times.

The get started image in particular, the rest seems fine 
image

**Comment by cleverActon0126** [comment]
Can you replace the image since this is what I can only get? Thanks.

**Comment by Erisa** [comment]
@goldsolitude is this okay now?

File: wiki/joining-windows-insiders.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+# Joining the Windows Insider Program
+
+This article covers how to join the Windows Insider Program to get builds of Windows 11.
+
+## Joining the Windows Insider Program
+
+> You must be logged onto a Microsoft Account to be able to join the Windows Insider program. This guide is dedicated to Windows 10 only.
```
Comment (Erisa) [inline wiki/joining-windows-insiders.md:7]:
```suggestion
> You must be logged onto a Microsoft Account to be able to join the Windows Insider program.
> This guide is intented to be followed from Windows 10 only, but allows you to receieve Windows 11 builds after joining.
```

File: wiki/joining-windows-insiders.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+1. Open the Settings app. You can do this by searching for it in the Start menu, pressing <kbd>Win</kbd> + <kbd>I</kbd>, or clicking the Settings button found above the Power button in Start. After that, open the *Update & Security* Section. 
+
+<img src="./img/joining-windows-insider/settings-home.png" width="500px">
+
+2. Click on *Windows Insider Program* on the sidebar to navigate to the sign up page.
```
Comment (Erisa) [inline wiki/joining-windows-insiders.md:13]:
```suggestion
2. Click on **Windows Insider Program** on the sidebar to navigate to the sign up page.
```

File: wiki/joining-windows-insiders.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
+# Joining the Windows Insider Program
+
+This article covers how to join the Windows Insider Program to get builds of Windows 11.
+
+## Joining the Windows Insider Program
+
+> You must be logged onto a Microsoft Account to be able to join the Windows Insider program. This guide is dedicated to Windows 10 only.
+
+1. Open the Settings app. You can do this by searching for it in the Start menu, pressing <kbd>Win</kbd> + <kbd>I</kbd>, or clicking the Settings button found above the Power button in Start. After that, open the *Update & Security* Section.
```
Comment (Erisa) [inline wiki/joining-windows-insiders.md:9]:
```suggestion
1. Open the Settings app. You can do this by searching for it in the Start menu, pressing <kbd>Win</kbd> + <kbd>I</kbd>, or clicking the Settings button found above the Power button in Start. After that, open the **Update & Security** Section. 
```

File: wiki/joining-windows-insiders.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+1. Open the Settings app. You can do this by searching for it in the Start menu, pressing <kbd>Win</kbd> + <kbd>I</kbd>, or clicking the Settings button found above the Power button in Start. After that, open the *Update & Security* Section. 
+
+<img src="./img/joining-windows-insider/settings-home.png" width="500px">
+
+2. Click on *Windows Insider Program* on the sidebar to navigate to the sign up page.
+
+<img src="./img/joining-windows-insider/settings-select-side.png" width="500px">
+
+3. Click on the *Get Started* button to join the Windows Insider Program.
```
Comment (Erisa) [inline wiki/joining-windows-insiders.md:17]:
```suggestion
3. Click on the **Get Started** button to join the Windows Insider Program.
```

File: wiki/joining-windows-insiders.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+
+<img src="./img/joining-windows-insider/get-started.png" width="500px">
+
+4. Click on *Register* to join the Windows Insider Program.
+
+<img src="./img/joining-windows-insider/register.png" width="500px">
+
+5. You will be prompted with a pop-up as shown below, click on *Sign Up*.
```
Comment (Erisa) [inline wiki/joining-windows-insiders.md:25]:
```suggestion
5. You will be prompted with a pop-up as shown below, click on **Sign Up**.
```

File: wiki/joining-windows-insiders.md [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+
+<img src="./img/joining-windows-insider/signup.png" width="500px">
+
+6. Read the [Windows Insider Program Agreement](https://insider.windows.com/program-agreement) and the [Microsoft Insider Privacy Agreement](https://privacy.microsoft.com/privacystatement), then check *I've read and accept the terms of this agreement* if you agree. Then click *Submit*.
```
Comment (Erisa) [inline wiki/joining-windows-insiders.md:29]:
```suggestion
6. Read the [Windows Insider Program Agreement](https://insider.windows.com/program-agreement) and the [Microsoft Insider Privacy Agreement](https://privacy.microsoft.com/privacystatement), then check **I've read and accept the terms of this agreement** if you agree. Then click **Submit**.
```

File: wiki/joining-windows-insiders.md [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+
+<img src="./img/joining-windows-insider/link-an-account.png" width="500px">
+
+7. Link an account to join the Windows Insider Program, then click *Continue*.
+
+> You can choose any account to signup, but your personal Microsoft Account is recommmended.
+
+<img src="./img/joining-windows-insider/account-link.png" width="350px">
+
+8. Now select a channel to receive insider builds, then click *Confirm*. Here are some details on the details of the 3 channels:
```
Comment (Erisa) [inline wiki/joining-windows-insiders.md:43]:
```suggestion
8. Now select a channel to receive insider builds, then click **Confirm**. Here are some details on the details of the 3 channels:
```

File: wiki/joining-windows-insiders.md [hunk 8]
```diff
@@ -0,0 +1,129 @@
...
+
+6. Read the [Windows Insider Program Agreement](https://insider.windows.com/program-agreement) and the [Microsoft Insider Privacy Agreement](https://privacy.microsoft.com/privacystatement), then check *I've read and accept the terms of this agreement* if you agree. Then click *Submit*.
+
+<img src="./img/joining-windows-insider/agree-and-submit.png" width="500px">
+
+Shortly after that you should see a pop-up saying that you have successfully singed up for the Windows Insider Program. You now have to choose an account to signup as a Windows Insider.
+
+<img src="./img/joining-windows-insider/link-an-account.png" width="500px">
+
+7. Link an account to join the Windows Insider Program, then click *Continue*.
```
Comment (Erisa) [inline wiki/joining-windows-insiders.md:37]:
```suggestion
7. Link an account to join the Windows Insider Program, then click **Continue**.
```

File: wiki/joining-windows-insiders.md [hunk 9]
```diff
@@ -0,0 +1,129 @@
...
+| Dev Channel                | Ideal for highly technical users. Be the first to access the latest Windows 11 builds earliest in the development cycle with the newest code. There will be some rought edges and low stability.                                  |
+| Beta Channel (Recommended) | Ideal for early adopters. These Windows 11 builds will be more reliable than builds from our Dev Channel. woth updates validated by Microsoft. Your feedback has the greatest impact here.                                             |
+| Release Preview Channel    | Ideal if you want to preview and certain key features, plus get optional access to the next version of Windows 10 before it's generally available to the world. This channel is also recommended for commercial user.                  |
+
+<img src="./img/joining-windows-insider/channels.png" width="500px">
+
+> if you don't see all 3 channels, you might be limited by your hardware. 
+
+9. Read the [Microsoft Insider Privacy Statement](https://privacy.microsoft.com/privacystatement) and the [Microsoft Insider Program Agreement](https://insider.windows.com/program-agreement), then click *Confirm*.
```
Comment (Erisa) [inline wiki/joining-windows-insiders.md:55]:
```suggestion
9. Read the [Microsoft Insider Privacy Statement](https://privacy.microsoft.com/privacystatement) and the [Microsoft Insider Program Agreement](https://insider.windows.com/program-agreement), then click **Confirm**.
```

File: wiki/joining-windows-insiders.md [hunk 10]
```diff
@@ -0,0 +1,129 @@
...
+
+> Photo source from [Minitool](https://www.minitool.com/news/how-to-join-windows-insider-program.html) 
+
+Select *Optional diagnostic data* and return back the the Windows Insider Program page.
+
+<img src="./img/joining-windows-insider/optional-diagnos-select.png" width="500px">
+
+### Windows Insider Program Page not appearing
+
+A registry might be preventing the page from appearing in settings. Here is what you can do:
```
Comment (Erisa) [inline wiki/joining-windows-insiders.md:107]:
```suggestion
A registry key might be preventing the page from appearing in settings. Here is what you can do to fix it:

> Editing the registry incorrectly can cause major problems. Make sure you follow the instructions here exactly and don't proceed if anything is unusual.
```

File: wiki/joining-windows-insiders.md [hunk 11]
```diff
@@ -0,0 +1,129 @@
...
+
+<img src="./img/joining-windows-insider/run.png" width="300px">
+
+> You might be prompted with the User Account Control popup, click yes or enter an adminstrator's username and password to continue.
+
+2. Go to the following registry key: `HKEY_LOCAL_MACHINE/SOFTWARE/Microsoft/WindowsSelfHost/UI/Visibility`
+
+<img src="./img/joining-windows-insider/registry-1.png" width="500px">
+
+> If the registry does not exists, right click the key *UI* on the side bar and press *New > Key* and name it `Visibility`.
```
Comment (Erisa) [inline wiki/joining-windows-insiders.md:119]:
```suggestion
> If the registry key does not exist, right click the key `UI` on the side bar and press **New > Key** and name it `Visibility`.
```

File: wiki/joining-windows-insiders.md [hunk 12]
```diff
@@ -0,0 +1,129 @@
...
+
+3. Right click and create a new 32-bit DWORD value named `HideInsiderPage` on the right hand side. Set the value to `0`.
+
+<img src="./img/joining-windows-insider/registry-2.png" width="500px">
+
+You may now close the Registry Editor window and restart your computer. The Windows Insider Program page should appear after the restart.
+
+### Error Code 
+
+You may see an error code as you are trying to register to be a Windows Insider. There are a lot of different error codes, please search up the error code for solutions.
```
Comment (Erisa) [inline wiki/joining-windows-insiders.md:129]:
```suggestion
You may see an error code while trying to register to be a Windows Insider. There are a lot of different error codes, so you should search it on the Internet or ask in [our Discord server](https://discord.gg/microsoft) for support (`#insiders` channel).
```

File: wiki/joining-windows-insiders.md [hunk 13]
```diff
@@ -0,0 +1,129 @@
...
+
+<img src="./img/joining-windows-insider/stop-receiving-builds.png" width="500px">
+
+6. You should see *This email address is not registered as a Windows Insider*. If you don't see it, please redo step 3 and step 4.
+
+## Troubleshooting
+
+### Optional Diagnostics Data is diabled
+
+If you see *To manage the Windows Insider programme settings for your device and allow it to stay in the Windows Insider programme, you'll need to turn on optional diagnostic data.*, click on *Go to Diagnostics & Feedback settings to turn on optional diagnostic data.*.
```
Comment (Erisa) [inline wiki/joining-windows-insiders.md:95]:
```suggestion
If you see *"To manage the Windows Insider programme settings for your device and allow it to stay in the Windows Insider programme, you'll need to turn on optional diagnostic data."*, click on *"Go to Diagnostics & Feedback settings to turn on optional diagnostic data."*. 
```

File: wiki/joining-windows-insiders.md [hunk 14]
```diff
@@ -0,0 +1,129 @@
...
+
+<img src="./img/joining-windows-insider/leaving-the-program.png" width="500px">
+
+> If you see *You must login to leave the program*, login on the top right corner of the website.
+
+5. Click on *Stop receiving Insider Preview Builds*.
+
+<img src="./img/joining-windows-insider/stop-receiving-builds.png" width="500px">
+
+6. You should see *This email address is not registered as a Windows Insider*. If you don't see it, please redo step 3 and step 4.
```
Comment (Erisa) [inline wiki/joining-windows-insiders.md:89]:
```suggestion
6. You should see *"This email address is not registered as a Windows Insider"*. If you don't see it, please redo step 3 and step 4.
```

File: wiki/joining-windows-insiders.md [hunk 15]
```diff
@@ -0,0 +1,129 @@
...
+
+<img src="./img/joining-windows-insider/leaving-get-started.png" width="500px">
+
+4. You will be redicted to an official website. Click on *Leave the Program now*.
+
+<img src="./img/joining-windows-insider/leaving-the-program.png" width="500px">
+
+> If you see *You must login to leave the program*, login on the top right corner of the website.
+
+5. Click on *Stop receiving Insider Preview Builds*.
```
Comment (Erisa) [inline wiki/joining-windows-insiders.md:85]:
```suggestion
5. Click on **Stop receiving Insider Preview Builds**.
```

File: wiki/joining-windows-insiders.md [hunk 16]
```diff
@@ -0,0 +1,129 @@
...
+
+1. Open *Settings*, click on *Updates & Security*
+
+<img src="./img/joining-windows-insider/settings-home.png" width="500px">
+
+2. Click on *Windows Insider program*
+
+<img src="./img/joining-windows-insider/settings-select-side.png" width="500px">
+
+3. Click on *Leave the Insider Program*.
```
Comment (Erisa) [inline wiki/joining-windows-insiders.md:75]:
```suggestion
3. Click on **Leave the Insider Program**.
```

File: wiki/joining-windows-insiders.md [hunk 17]
```diff
@@ -0,0 +1,129 @@
...
+
+## Leaving the Windows Insider Program
+
+If you don't want to participate in the Windows Insider Program anymore, you can leave the program. Your computer will stay on the build that you are currently on until an official build that is newer than the build you have. You can also revert to an older build if you haven't deleted the Windows build and if you installed the build less than 10 days ago.
+
+1. Open *Settings*, click on *Updates & Security*
+
+<img src="./img/joining-windows-insider/settings-home.png" width="500px">
+
+2. Click on *Windows Insider program*
```
Comment (Erisa) [inline wiki/joining-windows-insiders.md:71]:
```suggestion
2. Click on **Windows Insider program**
```

File: wiki/joining-windows-insiders.md [hunk 18]
```diff
@@ -0,0 +1,129 @@
...
+
+### Optional Diagnostics Data is diabled
+
+If you see *To manage the Windows Insider programme settings for your device and allow it to stay in the Windows Insider programme, you'll need to turn on optional diagnostic data.*, click on *Go to Diagnostics & Feedback settings to turn on optional diagnostic data.*. 
+
+<img src="./img/joining-windows-insider/optional-diagnos.png" width="500px">
+
+> Photo source from [Minitool](https://www.minitool.com/news/how-to-join-windows-insider-program.html) 
+
+Select *Optional diagnostic data* and return back the the Windows Insider Program page.
```
Comment (Erisa) [inline wiki/joining-windows-insiders.md:101]:
```suggestion
Select **Optional diagnostic data** and return back the the Windows Insider Program page.
```

**Review by Erisa — CHANGES_REQUESTED** [review]
My changes are
- Change italics to bold for buttons/etc
- Use quotes around longer italicised lines
- Fix some typos
- Clarify that the registry is dangerous
- Other fixes/clarifications

I will commit these suggestions and look at merging later today, have to go now

File: wiki/joining-windows-insiders.md [hunk 19]
```diff
@@ -0,0 +1,132 @@
...
+
+2. Click on **Windows Insider Program** on the sidebar to navigate to the sign up page.
+
+<img src="./img/joining-windows-insider/settings-select-side.png" width="500px">
+
+3. Click on the **Get Started** button to join the Windows Insider Program.
+
+<img src="./img/joining-windows-insider/get-started.png" width="500px">
+
+4. Click on *Register* to join the Windows Insider Program.
```
Comment (Erisa) [inline wiki/joining-windows-insiders.md:22]:
```suggestion
4. Click on **Register** to join the Windows Insider Program.
```

**Review by Erisa — CHANGES_REQUESTED** [review]
One missing italic -> bold

**Pull request merged**